### PR TITLE
Update versioned-s3-artifacts.any

### PR DESCRIPTION
**Pull request by jbayer**
added more detail at the beginning of the example

### CODE REVIEW

File: docs/examples/versioned-s3-artifacts.any [hunk 1]
```diff
@@ -5,7 +5,13 @@
 \title{Versioned S3 Artifacts}{versioned-s3-artifacts}
 
 This document will guide you through a pipeline modeled on a fairly common
-real-world use case of pushing tested artifacts into S3 buckets.
+real-world use case of pushing tested artifacts into S3 buckets from git commits through a build and test lifecycle to a final artifact.
```
Comment (xoebus) [inline docs/examples/versioned-s3-artifacts.any]: `git` -> `Git`

File: docs/examples/versioned-s3-artifacts.any [hunk 2]
```diff
@@ -5,7 +5,13 @@
...
+
+The end-to-end scenario is to monitor a git repository for commits, when
```
Comment (xoebus) [inline docs/examples/versioned-s3-artifacts.any]: `git` -> `Git`

File: docs/examples/versioned-s3-artifacts.any [hunk 3]
```diff
@@ -5,7 +5,13 @@
...
+new commits are detected, run the unit tests.
+If the unit tests pass, then create a new release candidate artifact with automated versioning and store the artifact in an s3 bucket.
```
Comment (xoebus) [inline docs/examples/versioned-s3-artifacts.any]: `s3` -> `S3`

File: docs/examples/versioned-s3-artifacts.any [hunk 4]
```diff
@@ -5,7 +5,13 @@
...
+When a new release candidate is created, run integration tests.
+If the integration tests pass, create a final version with automated versioning, put the artifact in an s3 bucket and ship it.
```
Comment (xoebus) [inline docs/examples/versioned-s3-artifacts.any]: `s3` -> `S3`

**Comment by xoebus** [comment]
Thanks for submitting this! I left a few minor style comments inline. Could you also wrap the paragraphs to 80 characters like the rest of the document?

tyvm

**Comment by jbayer** [comment]
@xoebus ok, i think i updated correctly to match your formatting requests.

**Comment by xoebus** [comment]
ty

**Pull request merged**